specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2022-10-01-preview/dataprotection.json

```diff
@@ -4875,6 +5230,85 @@
       "title": "Day",
       "type": "object"
     },
+    "DeletedBackupInstance": {
+      "description": "Deleted Backup Instance",
+      "required": [
+        "dataSourceInfo",
+        "policyInfo"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupInstance"
+        }
+      ],
+      "properties": {
+        "deletionInfo": {
+          "$ref": "#/definitions/DeletionInfo",
+          "description": "Deletion info of Backup Instance",
+          "readOnly": true
+        }
+      }
+    },
+    "DeletedBackupInstanceResource": {
+      "description": "Deleted Backup Instance",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DppResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DeletedBackupInstance",
+          "description": "DeletedBackupInstanceResource properties"
+        }
+      }
+    },
+    "DeletedBackupInstanceResourceList": {
+      "description": "List of DeletedBackupInstance resources",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DppResourceList"
+        }
+      ],
+      "properties": {
+        "value": {
+          "description": "List of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeletedBackupInstanceResource"
+          }
+        }
+      }
+    },
+    "DeletionInfo": {
+      "description": "Deletion Info",
+      "type": "object",
+      "properties": {
+        "deletionTime": {
+          "description": "Specifies time of deletion",
+          "type": "string",
+          "readOnly": true
+        },
+        "billingEndDate": {
+          "description": "Specifies billing end date",
+          "type": "string",
+          "readOnly": true
+        },
+        "scheduledPurgeTime": {
+          "description": "Specifies purge time",
+          "type": "string",
+          "readOnly": true
+        },
+        "deleteActivityID": {
```

Comment:
>deleteActivityID [](http://example.com/codeflow?start=9&length=16)

deleteActivityId

Reply:
This is consistent with our last version - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2022-09-01-preview/dataprotection.json#L5335